cfme/configure/configuration/server_settings.py

```diff
@@ -0,0 +1,409 @@
+
+    @View.nested
+    class web_services(View):   # noqa
+        """ Class represents Server WebServices Form """
+
+        mode = BootstrapSelect(id='webservices_mode')
+        security = BootstrapSelect(id='webservices_security')
+
+    @View.nested
+    class logging_form(View):   # noqa
+        """ Class represents Server Logging Form """
+
+        log_level = BootstrapSelect(id='log_level')
+
+    @View.nested
+    class custom_support_url(View):   # noqa
+        """ Class represents Server Custom Support URL Form """
+
+        url = Input(name='custom_support_url')
+        description = Input(name='custom_support_url_description')
+
+    @property
+    def is_displayed(self):
+        return {
+            self.server.is_active and
+            self.title.text == 'Basic Information'
+        }
+
+
+class ServerInformation(Updateable, Pretty, Navigatable):
+    """ This class represents the Server tab in Server Settings
+    Kwargs:
+        All lower parameters by default set to None
+        BasicInformationForm:
+            company_name: Company name, default value in "My Company"
+            appliance_name: Appliance name.
+            appliance_zone: Appliance Zone.
+            time_zone: Time Zone.
+            locale: Locale used for users UI
+        ServerControlForm (Server Roles):
+            websocket, ems_metrics_coordinator, cockpit_ws, smartproxy,
+            storage_metrics_collector, database_operations, smartstate, event,
+            storage_inventory, storage_metrics_processor, web_services, automate,
+            rhn_mirror, database_synchronization, ems_operations, ems_metrics_collector,
+            reporting, ems_metrics_processor, scheduler, git_owner, user_interface,
+            embedded_ansible, storage_metrics_coordinator, ems_inventory,
+            vmdb_storage_bridge, notifier: set True/False to change the state
+        VWwareConsoleSupportForm:
+            console_type - Server console type
+        NTPServersForm:
+            ntp_server_1, ntp_server_2, ntp_server_3 - Set ntp server
+        SMTPServerForm:
+            host: SMTP Server host name
+            port: SMTP Server port
+            domain: E-mail domain
+            start_tls: Whether use StartTLS
+            ssl_verify: SSL Verification
+            auth: Authentication type
+            username: User name
+            password: User password
+            from_email: E-mail address to be used as the "From:"
+            test_email: Destination of the test-email.
+        WebServicesForm:
+            mode: web services mode
+            security: security type
+        LoggingForm:
+            log_level: log level type
+        CustomSupportURL:
+            url: custom url
+            decryption: url description
+    """
+    CONSOLE_TYPES = ('VNC', 'VMware VMRC Plugin', 'VMware WebMKS')
+
+    basic_information = ['company_name', 'appliance_name', 'appliance_zone', 'time_zone', 'locale']
```

Comment:
**REQUIRED**: If these are here for helpers to split up the expected form inputs, can we make them ```_basic_information``` etc so that they are not readily readable as attrs that other people may want to use

Reply:
Done